src/ActionList/Header.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import styled, {css} from 'styled-components'
+import {get} from '../constants'
+
+export interface HeaderProps extends React.ComponentPropsWithoutRef<'div'> {
+  variant: 'subtle' | 'filled'
+  title: string
+  auxiliaryText?: string
+}
+
+const StyledHeader = styled.div<{variant: HeaderProps['variant']}>`
+  padding: calc((${get('space.3')} - ${get('space.1')}) / 2) ${get('space.2')};
+  font-size: ${get('fontSizes.0')};
+  font-weight: ${get('fontWeights.bold')};
+  color: ${get('colors.text.secondary')};
+
+  ${({variant}) =>
+    variant === 'filled' &&
+    css`
+      background: ${get('colors.bg.gray')};
+      margin: ${get('space.2')} -${get('space.2')};
```

Comment:
Same comment regarding negative margin.

Reply:
This is adapted from [primer/overlay-prototype actionList.scss#L416](https://github.com/primer/overlay-prototype/blob/0b06227bd0ef15fa1e1ff967c01ce6289fe358f6/src/scss/actionList.scss#L416). cc @vdepizzol

Reply:
Fixed in https://github.com/primer/components/pull/1078/commits/1e44f5c64fdcfb3f75a6ce280fd5a43c6d7fee5d!